javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -45,18 +45,28 @@ class ExprOrType extends @exprortype, Documentable {
     )
   }
 
-  /** Gets this expression or type, with any surrounding parentheses removed. */
+  /**
+   * Gets this expression or type, with any surrounding parentheses removed.
+   *
+   * Also see `getUnderlyingValue` and `getUnderlyingReference`.
+   */
   ExprOrType stripParens() { result = this }
 
   /**
    * Gets the innermost reference that this expression evaluates to, if any.
+   * For example, `o.m` of `(o.m)`, but not of `(0, o.m)`.
```

Comment:
I'd prefer a more complete listing of all covered cases (also for `getUnderlyingValue`).

Reply:
Done.